### ISSUE TITLE
Questions about Performance on CASIA_WebFace

### ISSUE THREAD

**Issue by mls1999725** [IrvingMeng/MagFace#6]
Hi, I have tried to reproduce your idea in my codes, I just modified the part of softmax loss to MagLoss, but I got the accuray of 97.70%. Whether the results will be inconsistent on different datasets? The figure below is the result of the last few epochs.
![image](https://user-images.githubusercontent.com/43207078/114706207-44653400-9d5b-11eb-86f7-843974836084.png)

**Comment by IrvingMeng** [comment]
Can you describe more details about your training/test settings?

Do you mean that you train the model on CASIA while get worse performance on LFW (than the value in the paper)? If so, that is normal as the paper uses MS1Mv2 is much larger than CASIA. 
See Table 8 in [ArcFace](https://arxiv.org/abs/1801.07698) where performance is 26.64 if using CASIA while is 80.96 if using MS1Mv2.

**Comment by mls1999725** [comment]
Hi, I read Arcface and I found that performance on CASIA is worse than MS1Mv2 in Table 6 on both identification and verification tasks.&nbsp;


I set CASIA as train dataset and the style of annotation file is "id name class", the size of input is 112x112 which is normalized to [-1, 1] by substract 127.5 and devided by 128 as well as RandomHorizontalFlip,&nbsp;batchsize is 512, dimension before full connection layer is 512, weight decay is 0.0005, momentum is 0.9, initial learning rate is 0.1 which decreases&nbsp; by 0.1 when 18 epoch and 28 epoch, and the number of epochs is 30. By the way, I use sphere20 as backbone, maybe it is the main reason.


While testing, batchsize is 1, I use cosine distance as metric method and 10-fold cross validation used to select best threshold and accuracy.


Thanks for your reply a lot!



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "IrvingMeng/MagFace"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月14日(星期三) 晚上9:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [IrvingMeng/MagFace] Questions about Performance on CASIA_WebFace (#6)





 
Can you describe more details about your training/test settings?
 
Do you mean that you train the model on CASIA while get worse performance on LFW (than the value in the paper)? If so, that is normal as the paper uses MS1Mv2 is much larger than CASIA.
 See Table 8 in ArcFace where performance is 26.64 if using CASIA while is 80.96 if using MS1Mv2.
